_data/navbars/getting-started.yml

```diff
@@ -20,19 +27,15 @@ section:
       path: /getting-started/kubernetes/managed-public-cloud/iks
     - title: Microsoft Azure Kubernetes Service (AKS)
       path: /getting-started/kubernetes/managed-public-cloud/aks
-  - title: Installation
-    path: /getting-started/kubernetes/installation/
+  - title: Self-managed public cloud
+    path: /getting-started/kubernetes/self-managed-public-cloud/
     section:
-    - title: Introduction
-      path: /getting-started/kubernetes/installation/overview
-    - title: Installing Calico for policy and networking (recommended)
-      path: /getting-started/kubernetes/installation/calico
-    - title: Installing Calico for policy (advanced)
-      path: /getting-started/kubernetes/installation/other
-    - title: Enabling application layer policy for Istio
```

Comment:
I feel like we're deleting a bunch of stuff that 
a) we still need
b) still exists in the repo

Reply:
@bmckercher123 could you please confirm whether deleting these navbar links was intended? These were lines were removed in this PR https://github.com/projectcalico/calico/pull/3144/files
I thought it was but now I can't recall asking you...

Reply:
Per Casey, we moved all Istio docs to the new section under Security. This came about with operator install where we started to think of "what is baseline install". This was considered post-install option.

Reply:
All of the changes are part of Alex's reorg work that Casey and I are executing. The deletions are expected. We will QA the sections to ensure changes are correct.

Reply:
@spikecurtis I've chatted about this with @bmckercher123 . The content in these getting-started docs are covered by the new self-managed on-premises installation docs. The "enable ALP" doc is moving to Security. Eventually, these installation docs that are getting unlinked from the navbar will be removed entirely (removing them now would cause a cascade of changes in docs that link to these docs).

I'm going to dismiss your review for now, let me know if there's something we missed.

Reply:
It doesn't make any sense to me to unlink documents from the navbar until they are ready to be removed.

It also doesn't make sense to me to remove "enable ALP" from the navbar now when it will move to Security as some unspecified time in the future.  The navbar change should happen in the same PR that actually moves it. Removing it now means we don't have any discoverable document that explains how to enable ALP.

Reply:
I have a PR ready with the move for ALP waiting for Laurence to merge this PR. It also fixes broken links as a result.